src/main/groovy/me/champeau/gradle/JMHPlugin.groovy

```diff
@@ -127,6 +159,13 @@ class JMHPlugin implements Plugin<Project> {
             }
         }
 
+        /** Intellij scope hack */
+        project.tasks.create(name: 'addJmhToTestScope', type: Test) {
```

Comment:
Make IDEA treat JMH sources as a test src dir instead of main src dir.

Not very nice, but does the trick without manual IDE configuration.